berkeley-mobile/StudyPact/Profile/ProfileViewController.swift

```diff
@@ -445,3 +460,40 @@ extension ProfileViewController {
         dismiss(animated: true, completion: nil)
     }
 }
+
+extension ProfileViewController {
+    func validateFacebook(text: String) -> String {
+        if CharacterSet.decimalDigits.isSuperset(of: CharacterSet(charactersIn: text)) {
+            return text
+        }
```

Comment:
what is this for?

Reply:
Ensure that the phone number is all digits

Reply:
this is for the facebook username though